src/commands/init.js

```diff
@@ -298,7 +309,7 @@ function askForDirectory(resolve) {
         name: 'name',
         message: 'What do you want to name the directory?'
     }], ({ name }) => {
-        const directoryPath = path.join(process.cwd(), name);
+        const directoryPath = name.indexOf('/') === 0 ? name : path.join(process.cwd(), name);
```

Comment:
I guess that `process.cwd()` should be changed to `directory` since we now get it from the command object? 

I assume we are using `indexOf` here to check if this is a absolutePath so we can create the directory outside the current scope if we want. Maybe we should rewrite this to use the `getAbsolutePath` utility function that we have https://github.com/rocjs/roc/blob/master/src/helpers/index.js#L13 that would both make the code more robust and easier to parse.